Apps/WebClient/src/ClientApp/src/models/hospitalVisitTimelineEntry.ts

```diff
@@ -37,7 +37,7 @@ export default class HospitalVisitTimelineEntry extends TimelineEntry {
         this.endDateTime = new DateWrapper(model.endDateTime, {
             hasTime: true,
         });
-        this.provider = model.provider;
+        this.provider = model.provider.length > 0 ? model.provider[0] : "";
```

Comment:
By returning an.array now, this piece of code will have to be duplicated when doing Export Report for Hospital Visits.